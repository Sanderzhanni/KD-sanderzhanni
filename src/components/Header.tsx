import React from 'react';


const Header = (): React.ReactElement =>(
  <>
    <div className={"nav-container"}>
      <div className={"nav"}>
        <div>
          <img src={process.env.PUBLIC_URL + 'img/lhv.png'} alt="lhv logo" className={"lhv-logo"}/>
        </div>
        <div className={"nav-button"}>Sisene</div>
      </div>
    </div>
  </>
);

export default Header;

