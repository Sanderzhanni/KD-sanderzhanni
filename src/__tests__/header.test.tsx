import renderer from "react-test-renderer";
import {BrowserRouter} from "react-router-dom";
import Header from "../components/Header";
import React from "react";

describe("testing header component", () =>{
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Header/>
        </BrowserRouter>
      ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});