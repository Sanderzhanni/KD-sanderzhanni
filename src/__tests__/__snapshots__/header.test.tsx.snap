// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing header component renders correctly 1`] = `
<div
  className="nav-container"
>
  <div
    className="nav"
  >
    <div>
      <img
        alt="lhv logo"
        className="lhv-logo"
        src="img/lhv.png"
      />
    </div>
    <div
      className="nav-button"
    >
      Sisene
    </div>
  </div>
</div>
`;
